lib/web_ui/lib/src/engine/surface/recording_canvas.dart

```diff
@@ -19,6 +19,31 @@ double _measureBorderRadius(double x, double y) {
   return clampedX * clampedX + clampedY * clampedY;
 }
 
+class RawRecordingCanvas extends BitmapCanvas
```

Comment:
This is not a new class. I just moved it from `dart:ui` to `dart:_engine`. It should have never been in `dart:ui`.